misk-zookeeper/src/test/kotlin/misk/clustering/zookeeper/ZkTestModule.kt

```diff
@@ -9,11 +9,19 @@ import misk.config.AppName
 import misk.inject.KAbstractModule
 import misk.inject.keyOf
 import misk.logging.getLogger
+import misk.security.ssl.CertStoreConfig
+import misk.security.ssl.SslLoader.Companion.FORMAT_JKS
+import misk.security.ssl.TrustStoreConfig
 import javax.inject.Singleton
 
 internal class ZkTestModule : KAbstractModule() {
   override fun configure() {
-    bind<ZookeeperConfig>().toInstance(ZookeeperConfig(zk_connect = "127.0.0.1:$zkPortKey"))
+    val keystorePath = this::class.java.getResource("/zookeeper/keystore.jks").path
```

Comment:
you should use the `ResourceLoader` to get these. should be able to bind the `ZookeeperConfig` to a provider that uses the `ResourceLoader`

Reply:
Blerg, decided to leave this as-is since using the `ResourceLoader` to get the absolute path of a resource isn't really supported.

Reply:
ah, you don't need the contents